pages/your_tasks/github_zenodo.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Obtaining a DOI for software
-description: How to obtain a DOI via Zenodo for your code releases 
-contributors: ["Shoaib Sufi", "Daniel Garijo"] 
-page_id: zenodo_doi
-related_pages: []
+title: Publishing software from GitHub to Zenodo
```

Comment:
Ok. I didn't want to make it only around DOI because that's not the only advantage of publishing to Zenodo but I suppose you are right and that's the main motivation for people